src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,7 +15,8 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL =
+            "https://github.com/AY2122S1-CS2103T-W10-4/tp/blob/master/docs/UserGuide.md";
```

Comment:
Please change it to "https://ay2122s1-cs2103t-w10-4.github.io/tp/UserGuide.html#quick-start" instead.